### ISSUE TITLE
Unexpected Attribute kms_key_id in aws_codecommit_repository resource

### ISSUE THREAD

**Issue by tomburge** [hashicorp/vscode-terraform#1686]
### Extension Version

v2.29.3

### VS Code Version

Version: 1.86.0 (system setup)
Commit: 05047486b6df5eb8d44b2ecd70ea3bdf775fd937
Date: 2024-01-31T10:28:19.990Z
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.19045

### Operating System

Windows 10 Pro 22H2 19045.3996

### Terraform Version

Terraform v1.6.2 on windows_amd64

### Steps to Reproduce

1. Open vscode
2. Create a file named vpc.tf
3. Configure as follows: 
```
resource "aws_codecommit_repository" "repository" {
  repository_name = var.name
  default_branch  = var.default_branch
  description     = var.description
  kms_key_id      = var.kms_key_arn
  tags            = var.tags
}
```
4. Install and enable the official Hashicorp terraform for visual studio code.

### Expected Behavior

VS Code shouldn't have a warning for a supported attribute in Terraform.

### Actual Behavior

VS Code gives a warning for an unsupported attribute in Terraform.

Terraform deploys successfully with the kms_key_id attribute.

### Workarounds

None

### References

- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/codecommit_repository

### Help Wanted

- [ ] I'm interested in contributing a fix myself

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by asychev** [comment]
Should be https://github.com/hashicorp/vscode-terraform/issues/1681

**Comment by dbanck** [comment]
@tomburge We just released a new version [2.29.4](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.29.4). This updates the schema to the latest version and should fix the validation error. The update should appear automatically in VS Code.

--- 

In case you experience any different validation related bug, please do let us know through a new issue.
